changelog/unreleased/issue-2571

```diff
@@ -0,0 +1,14 @@
+Enhancement: Check contents of unchanged files
```

Comment:
Maybe "Enhancement: restic backup is now self-healing" ?

I'd put some more emphasis on the fact that the checks actually cause corrections of broken repos.

Reply:
What about "Self-heal missing file parts during backup of unchanged files"? I'm a bit undecided whether that wording might be interpreted as a much stronger guarantee than it is.

Reply:
Thanks a lot for your suggestions! I overworked the changelog.